packages/react-components/src/utils/HOC.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import {State, FieldJson} from '@aemforms/af-core';
+import { useRuleEngine,  useFormIntl } from '@aemforms/af-react-renderer';
+import sanitizeHTML from 'sanitize-html';
+
+const richTextString = (stringMsg = '') => {
+  const htmlProp = {__html : sanitizeHTML(stringMsg)};
+  return (<div dangerouslySetInnerHTML={htmlProp} /> );
+};
+
+const withRuleEngine = (WrappedComponent: any) => {
```

Comment:
Rename this file to withRuleEngine.
I already create this method for material-ui Components. you can use this https://github.com/adobe/aem-forms-headless-components/blob/material-ui/packages/material-ui-based-components/src/shared/withRuleEngine.tsx